std_msgs/Header header

string mode
yumi_controller/YumiPosture[] trajectory
